### PR TITLE
Added limited access app type to docs

### PR DESCRIPTION
**Pull request by knmeiss**
# Description

This added in the limited access app type, which is part of the new app creation flow. 

Fixes DDOC-375

## Checklist

- [ X] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own changes
- [ X] I have run `yarn lint` to make sure my changes pass all linters
- [ X] I have pulled the latest changes from the upstream developer branch

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

File: content/guides/applications/custom-apps/app-token-setup.md [hunk 1]
```diff
@@ -90,37 +78,38 @@ required.
 
 ### Primary and Secondary App Tokens
 
-Authentication in this kind of application is done through preconfigured [App
-Tokens][app-token]. To configure an app token, head over to the [developer
-console][devconsole], select your application, and select the "Configuration"
-panel from the left-hand sidebar.
+Authentication with Limited Access Apps is done through preconfigured [App
+Tokens][app-token]. To configure an app token, navigate to the **Configuration**
+tab for your application within the[Developer console][devconsole].
```
Comment (jcleblanc) [inline content/guides/applications/custom-apps/app-token-setup.md:83]: Add space before the Developer console link

File: content/guides/applications/custom-apps/jwt-setup.md [hunk 2]
```diff
@@ -112,59 +104,60 @@ openssl rsa -in private.pem -outform PEM -pubout -out public.pem
  Windows users can install and use the [Cygwin][cygwin] package to run OpenSSL.
 </Message>
 
-Next, head over to your application in the [developer console][devconsole] and
-select the"Configuration" link from the left sidebar in your application
-and scroll down to the "Add and Manage Public Keys" section.
+Then, navigate to configuration tab for your application in the
```
Comment (jcleblanc) [inline content/guides/applications/custom-apps/jwt-setup.md:107]: Recommend changing "navigate to configuration tab..." to "navigate to **the** configuration tab..."

File: content/guides/applications/custom-apps/jwt-setup.md [hunk 3]
```diff
@@ -112,59 +104,60 @@ openssl rsa -in private.pem -outform PEM -pubout -out public.pem
...
+[Developer console][devconsole]and scroll down to the
```
Comment (jcleblanc) [inline content/guides/applications/custom-apps/jwt-setup.md:108]: Add space after developer console link

File: content/guides/applications/custom-apps/jwt-setup.md [hunk 4]
```diff
@@ -112,59 +104,60 @@ openssl rsa -in private.pem -outform PEM -pubout -out public.pem
...
+### Re-authorization after making configuration changes
 
 When the application's scopes or access level change the application needs to be
 re-authorized. Repeat the process above and request a new Access Token for the
 new changes to take effect.
 
 ## Basic configuration
 
-Before the application can be used, some basic additional configuration might be
+Before the application can be used, some additional, basic configuration is
```
Comment (jcleblanc) [inline content/guides/applications/custom-apps/jwt-setup.md:143]: This reads a little strange to me (the old sentence does too). Perhaps just cut it down to "Before the application can be used, some additional configuration is required".

File: content/guides/applications/custom-skills/setup.md [hunk 5]
```diff
@@ -15,83 +15,80 @@ Setting up a Custom Skill is a multi-step process.
...
-
-To set up the Invocation URL, head over to the [Developer Console][devconsole],
-select your application, and select on the "Configuration" panel on the left
-hand side.
+machine, or serverless function. The only requirement is that the URL is 
+publicly available and accessible by Box servers. For this reason, `localhost` 
+is not a valid address.
 
-Scroll down to the "Invocation URL" section.
+To set up the Invocation URL, navigate to the ** Configuration** tab of the
```
Comment (jcleblanc) [inline content/guides/applications/custom-skills/setup.md:74]: Remove the space in ** Configuration** - looks like it's not resolving into bold text.

File: content/guides/applications/custom-skills/setup.md [hunk 6]
```diff
@@ -15,83 +15,80 @@ Setting up a Custom Skill is a multi-step process.
...
+[Developer Console][devconsole] and scroll down to the "Invocation URL" section.
 
 <ImageFrame border width="600" center>
 
 ![App name form](../images/app-invocation-url.png)
 
 </ImageFrame>
 
-Fill in a secure HTTPs address and save the form. The invocation URL has now
+Fill in a secure HTTPs address and save the form. The invocation URL is now
```
Comment (jcleblanc) [inline content/guides/applications/custom-skills/setup.md:83]: Two items: Let's capitalize HTTPS for consistency (HTTPs -> HTTPS) and for the last sentence, recommend changing it to "The invocation URL is now configured." (drop **been**)

**Review by jcleblanc — CHANGES_REQUESTED** [review]
Just a few small tweaks, but otherwise lgtm

File: content/guides/applications/custom-skills/setup.md [hunk 7]
```diff
@@ -15,83 +15,80 @@ Setting up a Custom Skill is a multi-step process.
...
 ### 3. Provide a name
 
-On the next screen, provide a unique name for your application. This name needs
-to be unique across all applications on Box.
+Finally, select a unique name for your application and click **Create App**.
 
 <ImageFrame border width="600" center>
 
-![App name form](../images/app-name.png)
+![App name form](../images/skill-name.png)
```
[the remark this answers is not in the document]
Comment (knmeiss) [reply content/guides/applications/custom-skills/setup.md:46]: moved file so I could keep this url

**Review by jcleblanc — APPROVED** [review]

**Pull request merged**